### ISSUE TITLE
double registration of ResolverVer1_2

### ISSUE THREAD

**Issue by RafikTSG** [playgameservices/play-games-plugin-for-unity#1361]
Unity: 5.3.4f1
GPG: GooglePlayGames.PluginVersion.VersionString = "0.9.34"

Curiously there is double registration of resolver inside file Assets/PlayServicesResolver/Editor/ResolverVer1_2.cs 

```
        static ResolverVer1_2()
        {
            PlayServicesResolver.RegisterResolver(new ResolverVer1_2());
            Debug.Log("Registering resolver version " + MajorVersion + "." +
                MinorVersion + "." + PointVersion);
            PlayServicesResolver.RegisterResolver(new ResolverVer1_2());

        }
```
Looks like mistake...or maybe not?

**Comment by kruncher** [comment]
Also I am seeing a lot of spam being logged to the console regarding the resolver version:

![image](https://cloud.githubusercontent.com/assets/554215/18774960/1532856e-8156-11e6-9d5e-74b301bbcfa2.png)

**Comment by claywilkinson** [comment]
Thanks for the feedback - the next release will improve the resolver.

**Comment by claywilkinson** [comment]
Fixed in 0.9.35
